OSDOCS#17173: CQA 2.0 cert-manager Operator (#104809)
openshift/openshift-docs: modules/cert-manager-configuring-routes.adoc

[id="cert-manager-configuring-routes_{context}"]
= Configuring certificates to secure routes in your cluster

[role="_abstract"]

> The following steps demonstrate the process of utilizing the {cert-manager-operator} with the _Let's Encrypt_ ACME HTTP-01 challenge type to secure the route resources in your {product-title} cluster.

Reviewer: According to the SSG, we're supposed to avoid self-referential language such as "The following steps..." in the short description. See https://redhat-documentation.github.io/supplementary-style-guide/#shortdesc

```suggestion
You can secure the route resources in your {product-title} cluster by utilizing the {cert-manager-operator} with the _Let's Encrypt_ ACME HTTP-01 challenge type.
```
